docs/google/engine.md

```diff
@@ -0,0 +1,95 @@
+# Quantum Engine
+
+The Quantum Engine, via the `cirq.google.Engine` class, executes programs and jobs using the
+Quantum Engine API. 
+
+Note that the Quantum Engine API is not yet open for public access.
+
+## Authenticating to Google Cloud
+
+Before you begin, you will need to create a Google Cloud project with the API enabled
+and billing enabled.  You will then to create credentials in order to access the API.
+Information 
+
+`gcloud auth application-default login`
+
+From a colab, you can execute:
+
+```
+from google.colab import auth
+auth.authenticate_user(clear_output=False)
+```
+
+More information on creating application default credentials can be found on the
+[Google cloud](https://cloud.google.com/docs/authentication/production) website.
+
+## Engine class
+
+The `Engine` class is the entry point to communicate with the API.
+
+It can be initialized using your project id (found within your cloud console).
```

Comment:
UX guidance is to use Google Cloud Platform to refer to the console UI on first use and "Cloud Platform" on subsequent use.

Reply:
Changed to "Google Cloud Platform Console",  Is that what you mean, or should it just be "Google Cloud Platform"?